Sources/Mistica/Components/Tag/TagView.swift

```diff
@@ -137,6 +137,15 @@ public class TagView: UIView {
         }
     }
 
+    public var labelAccessibilityIdentifier: String? {
+        get {
+            label.accessibilityIdentifier
+        }
+        set {
+            label.accessibilityIdentifier = newValue
+        }
+    }
```

Comment:
Here you should override accesibilityIdentifier

```suggestion
    override public var accessibilityIdentifier: String? {
        get {
            label.accessibilityIdentifier
        }
        set {
            label.accessibilityIdentifier = newValue
        }
    }
```

Reply:
AFAIK, we agreed to give default value to Mistica components but also allow to customize them as it is done in Android.

Reply:
Not sure if would be the case were tagview needs to have a different Id than its label. In SW they use this ids for automated testing

Reply:
IMO the tag does not need a custom id because we can set a default id based on the style but if we need to set a custom id, I would used my approach overriding the accessibilityIdentifier of the UIView.

Reply:
done